### ISSUE TITLE
When maximizing a screen, apps have a hollow border

### ISSUE THREAD

**Issue by thiagodioniziofluxxer** [rxhanson/Rectangle#1474]
- When maximizing a screen, apps have a hollow border
![Captura de Tela 2024-09-20 às 08 14 50 (2)](https://github.com/user-attachments/assets/b6c02251-2fe7-4057-a110-6a0a6bd8f917)

macOS version: 15.0 (24A335)
Rectangle version: 0.83(89)

**Comment by rxhanson** [comment]
Do you have a window gap specified in the Gear tab of the settings window?

**Comment by bmmmm** [comment]
could that be the reason? 
![image](https://github.com/user-attachments/assets/c2dbd7b5-61cc-4823-a08d-bef6077f234c)

**Comment by Sangoku** [comment]
> could that be the reason? ![image](https://private-user-images.githubusercontent.com/24591225/372255948-c2dbd7b5-61cc-4823-a08d-bef6077f234c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIwOTcxNzgsIm5iZiI6MTczMjA5Njg3OCwicGF0aCI6Ii8yNDU5MTIyNS8zNzIyNTU5NDgtYzJkYmQ3YjUtNjFjYy00ODIzLWEwOGQtYmVmNjA3N2YyMzRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIwVDEwMDExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUyNzBjNjhlN2M5ZjEwN2U1YjVjZTkyZDExYzE3NjYzNTIyZjkzNzlhYWYzODAwOTg3NmY5YjJlNzI4MzIyNmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1zCRzptsG4RisbYNd52g_IXQ9wSujJhbyqRVg8eiiBs)

yes thanks. This is the cause. Turning this of makes it so all windows are same size. In my case one was full and one with margin.

**Comment by rxhanson** [comment]
Closing this one out since it looks like this is an uncommon edge case with macOS tiling conflict, but feel free to comment further if that's not the case.
